components/modal/modal.tsx

```diff
@@ -0,0 +1,64 @@
+import { Pencil, TrashAlt } from "@styled-icons/boxicons-regular";
+import { ArrowClockwise } from "@styled-icons/fluentui-system-filled";
+import { CalendarRtl } from "@styled-icons/fluentui-system-regular";
+import { SwitchVertical } from "@styled-icons/heroicons-outline";
+import React, { useState } from "react";
+import * as S from "./modal.styled";
+
+export const Modal = () => {
+  const [isModalOpen, setModalOpen] = useState<boolean>(true);
```

Comment:
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(true);
```
기본값으로 true 혹은 false 를 넣으면 자동으로 <boolean>으로 됩니다!